frontend/smart-kickers-game/src/components/Game/GameHistory/GameHistory.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import './GameHistory.css';
+import { TeamID } from '../../../constants/score.js';
+
+function GameHistory({ goalsArray }) {
+  return (
+    <div>
+      <h2>
+        <em>Game History</em>
+      </h2>
+      <div className="table-with-stats history">
+        <div className="table-item">
+          <FontAwesomeIcon className="blue-team-icon" icon="fa-person" />
+          Blue
+        </div>
+        <div className="table-item">
+          <FontAwesomeIcon className="white-team-icon" icon="fa-person" />
+          White
+        </div>
+        {goalsArray.map((item) =>
+          item.teamID === TeamID.Team_blue ? (
+            <>
+              <div key={item} className="table-item">
+                {item.timestamp}
+              </div>
+              <div key={item + item.timestamp} className="table-item"></div>
```

Comment:
This `key` evaluates to (for example) "[Object object]time 0:3". I don't think `[Object object]` part should be there.

Reply:
What should we use instead? Is the first key `item` correct? Using `item.teamID + item.timestamp` gives us the warning that _each child in a list should have a unique "key"_.

Reply:
What about `<React.Fragment key={item.teamID + item.timestamp}>` instead of `<>`?

Reply:
Thanks, that works fine! We didn't know about it 🦫.